index.html

```diff
@@ -0,0 +1,191 @@
+                            <h1>About <br> Myself</h1>
+                        </div>
+                        <div class="section5-p">
+                            <p>Hello I’m a software developer! I can help <br> you build a product , feature or website <br> Look through some of my work and <br> experience! If you like what you see and <br> have a project you need coded, don’t <br> hestiate to contact me.</p>
+                        </div>
+                        <h3 class="header">LET'S CONNECT</h3>
+                    <div class="icons-container">
+                        <ul>
+                        <li><a href=""><img class="social-icons" src="images/twitter.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/medium.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/github.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/angelist.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/angellist.png" alt=""></a></li>
+                    </ul>
+                    </div>
+                <div class="resume">
+                    <img src="images/resume.png" alt="">
+                </div>
+                              
+                <div class="section5-head">
+                    <h1>Languages</h1>
+                </div>
+                <div class="lang">
+                    <img class="lango" src="images/down-arrow.png" alt="">
+                </div>  
+                <div class="languages">
+                    <div class="language-icon1">
+                        <img src="images/js.png" alt="">
+                        <a href=""><h4 class="icon1-head">javascript</h4></a>
+                    </div>
+                    <div class="language-icon2">
+                        <img src="images/html-new.png" alt="">
+                        <a href=""><h4 class="icon1-head">HTML</h4></a>
+                    </div>
+                    <div class="language-icon3">
+                        <img src="images/css-new.png" alt="">
+                        <a href=""><h4 class="icon1-head">CSS</h4></a>
+                    </div>
+                </div>   
+                <div class="section5-head2">
+                    <h1>Frameworks</h1>
+                    <img class="arrow-next1" src= "images/next.png" alt="">
+                </div>
+                <div class="section5-head3">
+                    <h1>Skills</h1>
+                    <img class="arrow-next2" src="images/next.png" alt="">
+                </div>
+
+<section class="first-section">
+    <h2>Contact Me</h2>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/mjvzkobj" method="post">
```

Comment:
- _**kindly fix those design issues to match Figma design**_ :

- [ ] The Button styling doesn't match figma
- [ ] the fon used in the input doesn't match figma
- [ ] there are missing background image for the section
- [ ] the title color `contact me` should match figma as well

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/188007625-e92b591e-2563-49c9-9e56-87d795b2ca81.png)  | ![image](https://user-images.githubusercontent.com/89282221/188008051-8c07c266-d291-4c4b-8131-8d04e23bb037.png)